src/components/newListButton.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import getToken from '../lib/tokenGenerator';
+import * as ls from 'local-storage';
+var NewListButton = props => {
+	const onClick = event => {
```

Comment:
`var` works 100%. I would use `const` instead as most teams have adopted ES6 syntax.

Reply:
You might want to follow the file naming conventions that have been set up by the team and call this file `NewListButton` instead. Have you thought about just using `react-router-dom` `Link` instead of an `a` tag? 
Like `<Link to='/ShoppingList'`. Another thing to think about, naming-wise. Is this really a button?